### PR TITLE
Memphis final project

### PR DESCRIPTION
**Pull request by memphis88**

### CODE REVIEW

File: memphis88/project/backend/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+var fs = require('fs');
```
Comment (hulvei3) [inline memphis88/project/backend/quickstart.js]: you properly want to get this out of here :)

Comment (memphis88) [inline memphis88/project/backend/quickstart.js]: Actually this file is good to go in my next push, was using it as a guide. :)

**Comment by hulvei3** [comment]
You have created a very readable and neat piece of work here. Both server and client-side seems well-structured and easy to navigate in. I only took a superficial look, but it look good. Looking forward to see it - Great wrok

File: memphis88/project/backend/quickstart.js (hunk 1 above)
Comment (hulvei3) [inline memphis88/project/backend/quickstart.js]: Yes, I know that guide :)

**Pull request merged**